webapp-django/crashstats/documentation/jinja2/documentation/memory_dump_access.html

```diff
@@ -0,0 +1,49 @@
+{% extends "documentation/docs_base.html" %}
+
+{% block doc_title %}Memory Dump Access{% endblock %}
+
+{% block doc_content %}
+<div class="body">
+    <h1 id="policy">Access to memory dumps</h1>
+    <p>
+        Crash reports include memory dumps which contain sensitive information
+        and is not publicly available on Crash Stats. Memory dumps are important
+        for diagnosing and debugging issues. As such, users can request access
+        to memory dumps in order to debug issues.
+    </p>
+    <p>
+        Anyone with access to memory dumps and private data agrees to the following:
+    </p>
+    <pre>* Crash dumps contain memory contents, and may contain private user data
+* Only access dumps as necessary
+* Keep data securely on Mozilla hardware
+* When finished with a dump, delete it (within 30 days)
+* Do not share personal data from the dump publicly (on bugzilla, irc, etc)
+* Do not share personal data with non-employees, including partner organizations, without
+  consultation and explicit permission from Lonnen (lonnen@mozilla.com); a signed NDA and
+  other arrangements will be required of the other organization
+* It is fine to share stack traces, analysis, and other non-personal data
+* Access is conditional on employment and will be revoked upon departure from Mozilla
+* Access will be removed after a year of inactivity
+
+The most recent copy of this agreement is at: https://crash-stats.mozilla.com/documentation/pii_access/</pre>
```

Comment:
This agreement text exists in Mana. This moves it to be in the Crash Stats docs where it's easier to access.